library/group.inc

```diff
@@ -0,0 +1,89 @@
+<?php
+/** group.inc includes methods for groups  */
+
+require_once("{$GLOBALS['srcdir']}/sql.inc");
+require_once("{$GLOBALS['include_root']}/therapy_groups/therapy_groups_models/therapy_groups_model.php");
+require_once("{$GLOBALS['include_root']}/therapy_groups/therapy_groups_models/group_statuses_model.php");
+require_once("{$GLOBALS['include_root']}/therapy_groups/therapy_groups_models/therapy_groups_counselors_model.php");
+require_once("{$GLOBALS['include_root']}/therapy_groups/therapy_groups_models/users_model.php");
+require_once("{$GLOBALS['include_root']}/therapy_groups/therapy_groups_models/therapy_groups_participants_model.php");
+require_once("{$GLOBALS['include_root']}/therapy_groups/therapy_groups_controllers/therapy_groups_controller.php");
```

Comment:
Since this is a library, don't rely on globals for paths. instead use dirname(__FILE__). Here are several examples of use:
https://github.com/openemr/openemr/blob/master/library/sql.inc#L23-L26